docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
-            "address":"0x5259E44670053491E7b4FE4A120C70be1eAD646b"
-        }],
-        "issuedAt": "2022-10-30T08:27:01.667Z",
-        "expiration": "2022-10-30T08:32:01.667Z",
-        "nodeAddress": "https://node2.litgateway.com:7370"
-    }',
-    "address": "6a1f1e8a00b61867b85eaf329d6fdf855220ac3e32f44ec13e4db0dd303dea6a",
-    "algo": "ed25519"
-}
-```
+Usage of the `datil` and `datil-test` networks require the use of [Lit Capacity Credits](../../../sdk/capacity-credits.md). Currently, Capacity Credits are paid for using the `tstLPX` token and don't require any real-world money. However, in the future you will need to pay real-world money for usage of Lit networks, and `datil` and `datil-test` are the Lit networks where this functionality is being tested and refined.
 
-<br/>
+To implement payments correctly, include a [`capacityDelegationAuthSig`](https://developer.litprotocol.com/sdk/capacity-credits#createcapacitydelegationauthsig) within the `capabilityAuthSigs` array when generating session signatures.
 
-Here is what each field means:
+## Storing `SessionSigs`
 
-- `sig` is the signature produced by the ed25519 keypair signing the `signedMessage` payload
-- `derivedVia` should be `litSessionSignViaNacl` and specifies that the SessionSig object was created via the `NaCl` library.
-- `signedMessage` is the payload that was signed by the session keypair.
-- `address` is the session keypair public key.
-- `algo` is the signing algorithm used to generate the session signature.
+When running code to generate session signatures, you can store the session signatures and session keys using `LocalStorage` from the `node-localstorage` package.
 
-### Signed Message
+```javascript
+import { LocalStorage } from "node-localstorage";
+```
 
-Here is what each field in `signedMessage` means:
+When running code within a browser, this import is not needed, as the session signatures and keys will be stored within the browser's local storage. However, when running this code in an environment such as Node.js where browser local storage is not available, the `LocalStorage` module can be used to provide file-based storage. 
 
-- `sessionKey` is the session keypair public key.
-- `resourceAbilityRequests` is a lit of abilities that the session key is requesting to perform against the specified Lit resources during authentication. Read more [here](resources-and-abilities) about Lit Resources and Abilities.
-- `capabilities` is an array of one or more AuthSigs.
-- `issuedAt` is the time the SessionSig was issued.
-- `expiration` is the time the SessionSig becomes invalid.
-- `nodeAddress` is the specific URL the SessionSig is meant for.
+```javascript
+litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.DatilDev,
+    // This storageProvider object can be omitted if executing in a browser
+    storageProvider: {
+        provider: new LocalStorage("./lit_storage.db"),
+    },
+});
+```
```

Comment:
Consider:

When generating Session Signatures, you can store the signatures and session keys for reuse until the expire. How you store them depends on your runtime environment:

#### Running in a Browser

Session data is automatically stored in the browser's local storage, and no additional configuration is needed.

#### Running in a Node.js (or Similar) Environment

A `storageProvider` needs to specified when creating an instance of the `LitNodeClient`. You decide how this implemented, but the following is an example of using `LocalStorage` from the `node-localstorage` package:

```ts
import { LocalStorage } from "node-localstorage";
import { LitNodeClient } from "@lit-protocol/lit-node-client";
import { LitNetwork } from "@lit-protocol/constants";

const litNodeClient = new LitNodeClient({
    litNetwork: LitNetwork.DatilDev,
    // This storageProvider object can be omitted if executing in a browser
    storageProvider: {
        provider: new LocalStorage("./lit_storage.db"),
    },
});
```